### PR TITLE
Alex/load wheel refactor

### PR DESCRIPTION
**Pull request by alexdelicato**
added loading bar which shows loading with background blurred when cache is too old and requests are made.

### CODE REVIEW

**Review by reteppeter — APPROVED** [review]
I don't like that you can't use the website when it is loading the new version, but everything here is functional and useful in other loading scenarios.

**Comment by leanne-b** [comment]
image
1) Is there a reason behind positioning the text in the top left hand side of the progress bar?
It's minor but in the finalised design its in the centre underneath the progress bar.

2) I think the thickness of the progress bar can be fatter. Maybe like double or whatever seems reasonable. 

3) I think the background can be blurred out more or more opaque because I can see values and listed elements already in it behind the progress bar so it makes me think everything is complete although the progress bar is saying otherwise. 

4) Also I did not realise how big 20px font maybe we can make it 16px instead? Just a suggestion.

File: components/DependenciesContainer.tsx [hunk 1]
```diff
@@ -58,8 +58,8 @@ export default function DependenciesContainer(props: {
             <span>Filter</span>
           </button> */}
           <button onClick={callRefresh}>
-          <Image src={refreshIcon} alt="refresh" width="20px" height="20px"></Image>
-          <span>Refresh</span>
+            <Image src={refreshIcon} alt="refresh" width="20px" height="20px"></Image>
```
Comment (leanne-b) [inline components/DependenciesContainer.tsx:61]: mb change px -> rem

File: components/SummaryContainer.module.css [hunk 2]
```diff
@@ -102,6 +109,8 @@ div.summaryStyle {
     align-items: center;
     justify-content: center;
     max-height: 100%;
+    padding-top: 10px;
```
Comment (leanne-b) [inline components/SummaryContainer.module.css:112]: probs change to rem and add in-line comment of what it is in px

**Pull request merged**